### PR TITLE
CON-2668 [CHECKPOINT-RESTRUCTURE] Update `firstword` subject to require a function instead of a program

### PR DESCRIPTION
**Pull request by nprimo**
[CON-2667](https://01talent.atlassian.net/browse/CON-2667)

[CON-2667]: https://01talent.atlassian.net/browse/CON-2667?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by MSilva95 — APPROVED** [review]
LGTM!

File: subjects/firstword/README.md [hunk 1]
```diff
@@ -2,22 +2,38 @@
...
 ### Usage
 
+Here is a possible way to test your function:
+
+```go
+package main
+
+import "fmt"
+
+fmt.Print(FirstWord("hello there"))
```
Comment (MSilva95) [inline subjects/firstword/README.md:26]: missing `func main(){}` and the package piscine as it is in the lasword exercise

**Review by MSilva95 — CHANGES_REQUESTED** [review]

File: subjects/firstword/README.md (hunk 1 above)
Comment (nprimo) [reply subjects/firstword/README.md:26]: Thank you for the catch :pray:

**Review by HarryVasanth — APPROVED** [review]
> [!NOTE]
>  ✅ Production and pre-production prerequisites have been deployed.
> #### Clear to merge!

**Pull request merged**